services/spar/src/Spar/Sem/Random.hs

```diff
@@ -0,0 +1,14 @@
+module Spar.Sem.Random where
```

Comment:
What about this one?  https://hackage.haskell.org/package/polysemy-zoo-0.7.0.1/docs/Polysemy-Random.html

I think this is fine for now, but I don't know if I like the abstraction.  I would have made a more datatype-oblivious effect and then written application logic that uses that, and implements the functions below.

The way it looks now, I have to pull in the effect for generating scim tokens when i only want bytes for something completely unrelated.

I suppose you did this because it's easy to do and revisit and understand later?  I'm happy with keeping it as is, just checking :)

Reply:
In general, I think application-level effects should be as specific as possible. There's not much value in writing a really abstract effect, just to immediately specialize it. I'd be sympathetic to the argument that maybe we should expose these as `Input UUID`, or something, but I'm not entirely sure what it would buy us.